python/model_hosting_container_standards/common/transforms/utils.py

```diff
@@ -26,14 +28,40 @@ def _compile_jmespath_expressions(shape: Dict[str, Any]) -> Dict[str, Any]:
     return compiled_shape
 
 
-def set_value(obj: dict, path: str, value: Any) -> dict:
-    """Set value in a nested dict using JMESPath for the parent path."""
+def set_value(
+    obj: dict,
+    path: str,
+    value: Any,
+    create_parent: bool = False,
+    max_create_depth: int = DEFAULT_MAX_DEPTH_TO_CREATE,
+) -> dict:
+    """Set value in a nested dict using JMESPath for the parent path.
```

Comment:
Let's expand the doc string a bit to include the limitations - can only create dictionary/set container. Lists and other container types are unsupported.